examples/tutorials/advanced/dataset_on_surface.py

```diff
@@ -0,0 +1,167 @@
+    # the shading
+    shading="+a0/270+ne0.6",
+    perspective=[157.5, 30],  # Azimuth and elevation for the 3-D plot
+    zsize="1.5c",
+    plane="+gdarkgray",
+    frame=True,
+)
+
+# Add colorbar for curstal age
+fig.colorbar(frame=["x+lseafloor crustal age", "y+lMyr"], position="+n")
+
+# Show figure
+fig.show()
+
+
+# %%
+# 2. Draping an image
+# -------------------
+#
+# In the second example, the flag of Europe is plotted on top of a topographic
+# map of northwest Europe. This example is modified from
+# :gmt-docs:`GMT example 32 </gallery/ex32.html>`_.
+
+# Define study area
+region_2d = [3, 9, 50, 54]  # [lon_min, lon_max, lat_min, lat_max]
+region_3d = [*region_2d, -10, 900]  # Append [z_min, z_max] to this list
+perspective = [157.5, 30]  # Define azimuth, elevation for the 3-D plot
+
+# Coordinates and names of cities
+cities = pd.DataFrame(
+    {
+        "longitude": [7.10, 4.35, 5.69],
+        "latitude": [50.73, 50.85, 50.85],
+        "name": ["Bonn", "Bruxelles", "Maastricht"],
+    }
+)
+
+# -----------------------------------------------------------------------------
+# Create new Figure instance
+fig = pygmt.Figure()
+
+# Download elevation grid for the study region with a resolution of 30
+# arc-seconds and pixel registration and load it into a xarray.DataArray
+grd_relief = pygmt.datasets.load_earth_relief(resolution="30s", region=region_2d)
+
+# Download a image of the flage of Europe using rasterio and load it into a
+# xarray.DataArray
+url_to_image = "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b7/Flag_of_Europe.svg/1000px-Flag_of_Europe.svg.png"
+with rasterio.open(url_to_image) as dataset:
+    data = dataset.read()
+    drapegrid = xr.DataArray(data, dims=("band", "y", "x"))
+
+# Set up a colormap with two colors for the EU flag: blue (0/51/153) for the
+# background (value 0 in the nedCDF file -> lower half of 0-255 range) and
+# yellow (255/204/0) for the stars (value 255 -> upper half)
+pygmt.makecpt(cmap="0/51/153,255/204/0", series=[0, 256, 128])  # [min, max, step]
+
+fig.grdview(
+    projection="M12c",  # Mercator projection with a width of 12 centimeters
+    region=region_3d,
+    grid=grd_relief,  # Use elevation grid for z values
+    drapegrid=drapegrid,  # Drap image grid for the EU flag on top
+    cmap=True,  # Use colormap defined for the EU flag
+    surftype="i",  # Create an image plot
+    # Use an illumination from the azimuthal directions 0° (north) and 270°
+    # (west) with a normalization via a cumulative Laplace distribution for
+    # the shading
+    shading="+a0/270+ne0.6",
+    perspective=perspective,
+    zsize="1c",
+    plane="+glightgray",
+    frame=True,
+)
+
+# -----------------------------------------------------------------------------
+# Plot water, broders, and shorelines on top
+fig.coast(
+    water="white@50",
+    borders="1/1p,lightgray",
+    shorelines="1/0.5p,gray30",
+    perspective=perspective,
```

Comment:
Instead of using `perspective=perspective` repeatedly, we can just use `perspective=True`.
```suggestion
    perspective=True,
```

Reply:
Included in commit https://github.com/GenericMappingTools/pygmt/pull/3316/commits/965a29e97466585864280776e00ef0c9721f5599.